app/resources/app-base.html.template

```diff
@@ -13,7 +13,12 @@
 {% block title %}{{block.super}}{% endblock title %}
 
 {% block logo %}
-  <a href="{{env.global_url}}"><img src="{{env.global_url}}/logo.png" alt=""></a>
+  <a href="{{env.global_url}}">
+    <div class="logo">
```

Comment:
Do you mind moving the logo change to a separate changeset? This particular one will need a bit more discussion, involving PM.